orienteer-core/src/main/java/org/orienteer/core/component/command/SaveODocumentCommand.java

```diff
@@ -35,7 +36,7 @@ public SaveODocumentCommand(
 	@Override
 	public void onClick(AjaxRequestTarget target) {
 		getModelObject().getRecord().save();
-		super.onClick(target);
+        setResponsePage(new ODocumentPage(getModelObject()).setModeObject(DisplayMode.VIEW));
```

Comment:
Why do you need this instead of previous realization? It was ajax but this one is not.